coderd/idpsync/organizations_test.go

```diff
@@ -19,6 +19,88 @@ import (
 	"github.com/coder/coder/v2/testutil"
 )
 
+func TestFromLegacySettings(t *testing.T) {
+	t.Parallel()
+
+	t.Run("AssignDefault,True", func(t *testing.T) {
+		t.Parallel()
+		legacy := `{
+   "Field":"groups",
+   "Mapping":{
+      "engineering":[
+         "10b2bd19-f5ca-4905-919f-bf02e95e3b6a"
+      ]
+   },
+   "AssignDefault":true
+}`
```

Comment:
nit: could we make the indentation here a bit nicer? maybe something like

```suggestion
		legacy := `
			{
				"Field":"groups",
				"Mapping":{
					"engineering":[
						"10b2bd19-f5ca-4905-919f-bf02e95e3b6a"
					]
				},
				"AssignDefault":true
			}
		`
```